docs/quick-start/README.md

```diff
@@ -0,0 +1,79 @@
+---
+navGroup: FlowFuse Self-Hosted
+navTitle: Quick Start
+navOrder: 1
+meta:
+   description: Quickly install FlowFuse in Docker or Kubernetes based environments.
+   tags:
+     - start
+     - quick start
+     - quickstart
+     - installation
+     - flowfuse
+     - docker
+     - trial license
+     - deployment models
+---
+
+# Quick Start Guide
+
+This guide will help you quickly set up and run FlowFuse platform using Docker and docker-compose.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+1. A domain name that you own and can configure DNS settings for (explained in step 1)
+2. Docker and Docker Compose installed on your system (either as a standalone binary or as docker plugin)
+
+## Step 1: Configure DNS
+
+Before running the FlowFuse, you need to configure your fully qualified domain name settings:
+
+1. Point your main domain (e.g., `yourdomain.com`) to your server's IP address (works with subdomain as well).
+2. Set up a wildcard DNS record (e.g., `*.yourdomain.com`) to point to your server's IP address.
```

Comment:
Should this explicitly mention an A record

Reply:
> Should this explicitly mention an A record

Added.